contracts/FlowServiceAccount.cdc

```diff
@@ -92,9 +97,19 @@ pub contract FlowServiceAccount {
 
     /// Returns true if the given address is permitted to create accounts, false otherwise
     pub fun isAccountCreator(_ address: Address): Bool {
+        if !self.isRestrictedAccountCreationEnabled() {
```

Comment:
Would it be possible to rename this to something like `accountCreationRestricted` or something like that? The current name is kind of unclear what it is describing